query/graphql/src/test/groovy/org/codice/ddf/admin/graphql/test/GraphQLTransformationTest.groovy

```diff
@@ -23,6 +24,8 @@ class GraphQLTransformationTest extends Specification {
 
     static STATUS_OK = 200
 
+    static STATUS_INVALID_REQUEST = 400
```

Comment:
❗️ Update this test to match the http response code changes that need to be made.